pkg/diagnose/diagnose.go

```diff
@@ -70,6 +70,9 @@ func RunCollect(cmd *cobra.Command, args []string) {
 	operatorPodSelector, _ := labels.Parse("noobaa-operator=deployment")
 	c.CollectPodLogs(operatorPodSelector)
 
+	endpointPodSelector, _ := labels.Parse("noobaa-s3=" + options.SystemName)
```

Comment:
will it catch all endpoints (auto-scale...)

Reply:
checked a system with 2 endpoints - both had this label